airmozilla/popcorn/tests/test_views.py

```diff
@@ -157,6 +157,47 @@ def test_popcorn_editor(self):
         response = self.client.get(url, {'slug': event.slug})
         eq_(response.status_code, 200)
 
+    def test_save_edit(self):
+        event = Event.objects.get(title='Test event')
+        event.template.name = 'this is a vid.ly video'
+        event.template.save()
+
+        event.template_environment = {'tag': 'abc123'}
+        event.save()
+
+        url = reverse('popcorn:save_edit')
+
+        self._login()
+
+        response = self.client.post(url, {
+            'slug': 'does_not_exist',
+        })
+        eq_(response.status_code, 400)
```

Comment:
It checks for the field in the request before it get's the event by the slug. I'll change to event.slug to be more clear